WalletWasabi.Fluent/Controls/CurrencyEntryBox.axaml.cs

```diff
@@ -308,9 +308,15 @@ public async void ModifiedPasteAsync()
 			return;
 		}
 
+		// Ignore paste if there are invalid characters
+		if (!RegexDecimalCharsOnly().IsMatch(text))
+		{
+			return;
+		}
```

Comment:
This results in inconsistency. Amount flyout pops for `0.1x` but I cannot paste it manually.

I think you can just remove this condition.

Reply:
@soosr added validation for ClipboardObserver (flyout) too.

Removing this validation is nACK IMO, because it would make things like `0.0001x0` be interpreted as `0.00010` which is not good IMO